binance_client.go

```diff
@@ -223,9 +223,16 @@ func (bc *Client) GetAccountTradeHistory(symbol, startTime, endTime string, limi
 	}
 	rr := req.WithHeader(apiKeyHeader, bc.apiKey).
 		WithParam("symbol", symbol).
-		WithParam("startTime", startTime).
-		WithParam("endTime", endTime).
 		WithParam("limit", strconv.FormatInt(limit, 10))
+	if startTime != "" {
```

Comment:
startTime and endTime is optional, set "" and it should work fine, limit on other side is required